SteamKit2/SteamKit2/Steam/WebAPI/WebAPI.cs

```diff
@@ -21,6 +21,33 @@ namespace SteamKit2
     /// </summary>
     public sealed class WebAPI
     {
+        /// <summary>
+        /// Represents a response to a WebAPI request.
+        /// </summary>
+        public class WebAPIResponse<TResponse>
+        {
+            /// <summary>
+            /// Gets the result of the request.
+            /// </summary>
+            public EResult Result { get; }
+
+            /// <summary>
+            /// Gets the body of the request.
+            /// </summary>
+            public TResponse Body { get; }
+
+            /// <summary>
+            /// Constructs <see cref="WebAPIResponse{TResponse}"/> object.
+            /// </summary>
+            /// <param name="body">The result.</param>
+            /// <param name="result">The body.</param>
+            internal WebAPIResponse( TResponse body, EResult result )
```

Comment:
I would reverse these two parameters, also noting that everywhere else they are already in opposite order with the result coming before the body, even in the constructor body.

Also, do we want to have any null guarantees or checks here?

Reply:
I don't think null check is necessary here, deserialize will fail.